contracts/adapters/FlexibleLeverageStrategyAdapter.sol

```diff
@@ -975,6 +1174,14 @@ contract FlexibleLeverageStrategyAdapter is BaseAdapter {
         }
     }
 
+    /**
+     * Update globalLastTradeTimestamp and exchangeLastTradeTimestamp values
```

Comment:
Add a bit more context on why we always need to update both. i.e. our assumption that global must always be >= exchange specific timestamp